.github/workflows/soroban-rpc.yml

```diff
@@ -109,7 +109,7 @@ jobs:
     env:
       SOROBAN_RPC_INTEGRATION_TESTS_ENABLED: true
       SOROBAN_RPC_INTEGRATION_TESTS_CAPTIVE_CORE_BIN: /usr/bin/stellar-core
-      PROTOCOL_20_CORE_DEBIAN_PKG_VERSION: 19.13.1-1469.76fed6ead.focal~soroban
+      PROTOCOL_20_CORE_DEBIAN_PKG_VERSION: 19.13.1-1481.3acf6dd26.focal
```

Comment:
anytime this ref to core is changed, the equivalent core git ref related to this build will be needed in e2e, i've started [#952](https://github.com/stellar/soroban-tools/pull/952) for a draft to keep e2e up to date with current release stretch.

maybe we agree to sync up e2e in soroban-tools on a wider, less frequent iteration or once e2e is positioned more as nightly test per https://github.com/stellar/system-test/issues/10, then `e2e.yml` can be removed from soroban-tools as part of pr-commit workflow test.